http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/app/WorkflowsAppInstall.scala

```diff
@@ -0,0 +1,146 @@
+
+      // Resolve application insights in Azure
+      applicationInsightsComponent <- azureApplicationInsightsService.getApplicationInsights(
+        params.landingZoneResources.applicationInsightsName,
+        params.cloudContext
+      )
+
+      // Storage container is required for Workflows app
+      storageContainer <- F.fromOption(
+        params.storageContainer,
+        AppCreationException("Storage container required for Workflows app", Some(ctx.traceId))
+      )
+
+      // Databases required for Workflows App
+      dbNames <- F.fromOption(toWorkflowsAppDatabaseNames(params.databaseNames),
+                              AppCreationException("Database names required for Workflows app", Some(ctx.traceId))
+      )
+
+      // Postgres server required for Workflows App
+      postgresServer <- F.fromOption(
+        params.landingZoneResources.postgresServer,
+        AppCreationException("Postgres server required for Workflows app", Some(ctx.traceId))
+      )
+
+      // Get the pet userToken
+      tokenOpt <- samDao.getCachedArbitraryPetAccessToken(params.app.auditInfo.creator)
+      userToken <- F.fromOption(
+        tokenOpt,
+        AppCreationException(s"Pet not found for user ${params.app.auditInfo.creator}", Some(ctx.traceId))
+      )
+
+      values =
+        List(
+          // azure resources configs
+          raw"config.resourceGroup=${params.cloudContext.managedResourceGroupName.value}",
+          raw"config.batchAccountKey=${batchAccount.getKeys().primary}",
+          raw"config.batchAccountName=${params.landingZoneResources.batchAccountName.value}",
+          raw"config.batchNodesSubnetId=${params.landingZoneResources.batchNodesSubnetName.value}",
+          raw"config.drsUrl=${drsConfig.url}",
+          raw"config.landingZoneId=${params.landingZoneResources.landingZoneId}",
+          raw"config.subscriptionId=${params.cloudContext.subscriptionId.value}",
+          raw"config.region=${params.landingZoneResources.region}",
+          raw"config.applicationInsightsConnectionString=${applicationInsightsComponent.connectionString()}",
+
+          // relay configs
+          raw"relay.path=${params.relayPath.renderString}",
+
+          // persistence configs
+          raw"persistence.storageResourceGroup=${params.cloudContext.managedResourceGroupName.value}",
+          raw"persistence.storageAccount=${params.landingZoneResources.storageAccountName.value}",
+          raw"persistence.blobContainer=${storageContainer.name.value}",
+          raw"persistence.leoAppInstanceName=${params.app.appName.value}",
+          raw"persistence.workspaceManager.url=${params.config.wsmConfig.uri.renderString}",
+          raw"persistence.workspaceManager.workspaceId=${params.workspaceId.value}",
+          raw"persistence.workspaceManager.containerResourceId=${storageContainer.resourceId.value.toString}",
+
+          // identity configs
+          raw"workloadIdentity.serviceAccountName=${params.ksaName.value}",
```

Comment:
Should workflows app also still have an identity name here as per https://github.com/DataBiosphere/leonardo/pull/3666/files#r1319924610? I don't see where the identity name is being used in the charts

Reply:
I didn't see where the workflows-app [cromwell config](https://github.com/broadinstitute/terra-helmfile/blob/master/charts/workflows-app/templates/cromwell-config.yaml) needs the managed identity name -- maybe because it's a read-only Cromwell? 

That said, no harm in Leo passing the value in case Workflows-app needs it for some reason.

Reply:
Correct, it's set in config to be passed to the TES that is running the jobs, so not needed in Workflows app.

Reply:
Ah, I didn't see https://github.com/broadinstitute/terra-helmfile/pull/4583 and related discussion. Probably not needed for workflows app then?